ssl/t1_lib.c

```diff
@@ -943,7 +943,7 @@ int ssl_cipher_disabled(SSL *s, const SSL_CIPHER *c, int op)
 
 static int tls_use_ticket(SSL *s)
 {
-    if (s->options & SSL_OP_NO_TICKET)
+    if (s->options & SSL_OP_NO_TICKET || SSL_IS_TLS13(s))
```

Comment:
yeah, but we tend to (not styleguide requirement) use those parens, so consider it.  don't care much what you do.